mysql-test/suite/binlog/t/flashback.test

```diff
@@ -364,6 +364,124 @@ FLUSH LOGS;
 
 DROP TABLE t1;
 
+--echo # < CASE 8 >
+--echo # Update join, Delete join
+--echo #
+
+create table school(
+        schoolID int not null default 0 primary key,
+        schoolName blob not null
+)ENGINE=InnoDB;
+create table class(
+        classID int not null default 0 primary key,
+        className blob not null,
+        schoolID int not null default 0
+)ENGINE=InnoDB;
+create table student(
+        stdID int not null default 0 primary key,
+        stdName blob not null,
+        classID int not null default 0,
+        schoolID int not null default 0
+)ENGINE=InnoDB;
+INSERT INTO school (schoolID,schoolName) VALUES (1,"S-a"),(2,"S-b");
```

Comment:
In case we conclude that the extra options are indeed needed, we will need to at least execute `mysqlbinlong`  with the new options here so that we are sure the options are available and behave properly, but I'd vote for a more elaborate version in which we also check if data is indeed filtered out based on the values within the new options.

Reply:
> In case we conclude that the extra options are indeed needed, we will need to at least execute `mysqlbinlong` with the new options here so that we are sure the options are available and behave properly, but I'd vote for a more elaborate version in which we also check if data is indeed filtered out based on the values within the new options.

Ok, I can add a test case, mysqlbinlog --flashback --databases --flashback-tables will print the detailed results to ensure that the filtering is ok.

Reply:
Like I explained in another thread, please don't introduce `--flashback-tables`. Let us extend
`--table` and `--database` instead.